pkg/machinery/config/types/v1alpha1/v1alpha1_validation.go

```diff
@@ -792,10 +794,14 @@ func (k *KubeletConfig) Validate() ([]string, error) {
 	return nil, result.ErrorOrNil()
 }
 
-// Validate kubelet configuration.
+// Validate etcd configuration.
 func (e *EtcdConfig) Validate() error {
 	var result *multierror.Error
 
+	if e.CA() == nil {
```

Comment:
I don't remember if it runs for workers? on workers there's no requirement on etcd... or whole etcd config is nil?

Reply:
yeh, it's nil by default, unless someone adds those fields manually, in that case the other existing checks also fail